src/norm_fit/norm_fit.c

```diff
@@ -237,31 +255,38 @@ int main(int argc, char **argv)
     //2.2=====================removes spurios peaks (not smoothed yet) as not peaks up to min_distance/2
     num_peD = SpuriousPeaks(hist, bins, nbins, amp, pos, min_distance, num_peD);
 
-    //2 number of modes is the number of remaining peaks after distance and spurious peak (if needed)
+    //3 ----number of modes is the number of remaining peaks after distance and spurious peak (if needed)
     num_modes = num_peD;
+    //3.1  ----estimate stds of filtered in remaining peaks=modes
+
+    for (k=0; k<num_modes; k++)
+    {
+        sds[k]=EstimateStd(hist,bins,nbins,pos[k],amp[k]);// std of a filtered mode
```

Comment:
yes, initially Irina was outputting this but it broke the code that 
plots the normal on the insert size histogram so I removed it.

Steven

On 02/06/16 10:18, dkj wrote:
>
> In src/norm_fit/norm_fit.c 
> <https://github.com/wtsi-npg/npg_qc/pull/339#discussion_r65508037>:
>
> >      num_modes = num_peD;
> > +    //3.1  ----estimate stds of filtered in remaining peaks=modes
> > +
> > +    for (k=0; k<num_modes; k++)
> > +    {
> > +        sds[k]=EstimateStd(hist,bins,nbins,pos[k],amp[k]);// std of a filtered mode
>
> Set but not used?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/wtsi-npg/npg_qc/pull/339/files/864c60f455df5eb97420ce7a423f714f68afefa0#r65508037>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ABfQJDwGddfeyRHxeyx2wgri6MiEIbqDks5qHp_bgaJpZM4Irivf>.
>




-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.